src/extensions/isolator/isolator.ts

```diff
@@ -79,14 +79,19 @@ export default class Isolator {
         return { id, value: c };
       })
     );
+    const before = await getPackageJSONInCapsules(capsules);
 
     await writeComponentsToCapsules(components, graph, capsules, capsuleList);
+    const after = await getPackageJSONInCapsules(capsules);
```

Comment:
I keep removing these because they introduce at least one bug that I am aware of: if you change your package manager, they will not be reinstalled. I don't think this is the right place for this caching. (there are likely more issues with this, it's too simplistic).

Reply:
@qballer

Reply:
Hey thanks for commenting on your day off. This is a problem, should be solved if we write the packageMannager in package.json.

Reply:
Aren't we planning on doing this through a config hash?

Reply:
the script execution. No specific plan for packageManager install path. I just want to have small merged if I get a PR going doing execution caching (my next task) I will try to see how these thing cooperate.